frontend/src/styles/styles.scss

```diff
@@ -0,0 +1,70 @@
+/* You can add global styles to this file, and also import other style files */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap");
```

Comment:
We don't need font inclusion from the HTML(above) as well as the CSS file

Reply:
ok removed